### PR TITLE
build(attributable-map-2): use fluid-tsc

### PR DESCRIPTION
**Pull request by DLehenbauer**
Semi-automatic PR to switch to fluid-tsc

### CODE REVIEW

File: experimental/dds/attributable-map/api-extractor-cjs.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../common/build/build-common/api-extractor-base.cjs.primary.json",
+	// CJS is actually secondary; so, no report.
```
Comment (tylerbutler) [inline experimental/dds/attributable-map/api-extractor-cjs.json:4]: Should we have a api-extractor-base.cjs.secondary.json config?

File: experimental/dds/attributable-map/package.json [hunk 2]
```diff
@@ -100,6 +142,18 @@
...
+			"build:docs": {
+				"dependsOn": [
+					"...",
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
+				],
+				"script": false
+			}
+		}
+	},
```
Comment (tylerbutler) [inline experimental/dds/attributable-map/package.json:156]: We should be able to get rid of these overrides in the per-package fluid-build config now. There was some blocker but I think was resolved. I'll follow up separately.

**Review by tylerbutler — APPROVED** [review]

File: experimental/dds/attributable-map/api-extractor-cjs.json (hunk 1 above)
Comment (DLehenbauer) [reply experimental/dds/attributable-map/api-extractor-cjs.json:4]: That makes sense.  We're 50 packages in, so we'll need to do a cleanup pass.  Now tracked by AB#7417.

File: experimental/dds/attributable-map/package.json (hunk 2 above)
Comment (DLehenbauer) [reply experimental/dds/attributable-map/package.json:156]: I'll try excluding them in my next PR.  (Cleanup of existing packages now tracked by AB#7418.)

**Pull request merged**